pkg/commands/git_test.go

```diff
@@ -1817,7 +1817,7 @@ func TestGitCommandDiscardOldFileChanges(t *testing.T) {
 			"test999.txt",
 			test.CreateMockCommand(t, []*test.CommandSwapper{
 				{
-					Expect:  "git rebase --interactive --autostash 123456^",
+					Expect:  "git rebase --interactive --autostash abcdef^",
```

Comment:
I believe this is actually a breaking change in the `DiscardOldFileChanges` method: We want to begin an interactive rebase at a specific commit, then make some changes to that commit, and then continue. Here we're modifying the wrong commit.

I think a sufficient fix here is to just remove the '^' from the call to `PrepareInteractiveRebaseCommand` and we'll have retained the original behaviour